src/calendar/tests/CalendarMonth.spec.js

```diff
@@ -0,0 +1,232 @@
+import React from 'react/addons';
+import CalendarMonth from '../CalendarMonth';
+import CalendarDate from '../CalendarDate';
+import moment from 'moment';
+import {} from 'moment-range';
+import _ from 'underscore';
+
+
+const TestUtils = React.addons.TestUtils;
+
+describe('The CalendarMonth Component', function () {
+
+  beforeEach(function () {
+
+    const getCalendarMonth = (props) => {
+
+      props = _.extend({
+        firstOfWeek: 0,
+        firstOfMonth: this.firstOfMonth,
+        enabledRange: moment.range(moment(), moment().add(3, 'years')),
+        dateComponent: CalendarDate,
+        disableNavigation: false,
+        dateRangesForDate: function () {
+          return {
+            count: function () {
+              return props.count || 1;
+            },
+            getIn: function (data) {
```

Comment:
I am not sure this would help much to be fair. The props that we are using are shown in a single contract, immediately accessible.

As a whole, I allow a level of repetition in my tests as I believe it helps getting the context.